zlib/config.go

```diff
@@ -0,0 +1,144 @@
+package zlib
+
+import (
+	"bufio"
+	"encoding/gob"
+	"github.com/jessevdk/go-flags"
+	log "github.com/sirupsen/logrus"
+	"os"
+	"strings"
+)
+
+type Option struct {
+	OutputFileName     string     `short:"o" long:"output-file" default:"-" description:"Output filename, use - for stdout" json:"out"`
+	InputFileName      string     `short:"i" long:"input-file" default:"-" description:"Input filename, use - for stdin" json:"in"`
+	Meta               string     `short:"m" long:"metadata-file" default:"-" description:"Metadata filename, use - for stdout" json:"meta"`
+	Log                string     `short:"l" long:"log-file" default:"-" description:"Log filename, use - for stdout" json:"log"`
```

Comment:
This should probably be named consistently with OutputFileName and InputFileName as MetadataFileName. The same for Log.